chapters/classes.tex

```diff
@@ -1500,6 +1500,12 @@ \subsection{Real Type}\label{real-type}
 \index{unbounded@\robustinline{unbounded}!attribute of \robustinline{Real}}%
 \index{stateSelect@\robustinline{stateSelect}!attribute of \robustinline{Real}}
 
+The default \lstinline!max! and \lstinline!max!-attributes are the minimum and maximum representable finite floating point number of the underlying type.
+Therefore they do not constrain the value.
```

Comment:
It would be worth pointing out that this still means that infinite values are outside the allowed range:
```suggestion
Therefore they only constrain the value to be finite, i.e., floating point representations of infinity are not allowed.
```

Reply:
I will reformulate - in my opinion the constraint to be an actual finite number is part of the type, and the default min/max do not restrict that further.